tests/integration/fixture/lemma_widget.html

```diff
@@ -35,11 +36,14 @@
         "isSaving":false,
         "lemmas": [
             {"value": "lemma1", "language":"en"},
-            {"value": "lemma2", "language":"fr"}
+            {"value": "lémma2", "language":"fr"},
+            {"value": "기본형", "language": "ko"},
+            {"value": "Лемма", "language": "ru"},
+            {"value": "لليما", "language": "ar"}
         ]
     }
 </script>
 <div id="result">
 	<!-- generated by `npm run-script populate-fixtures` -->
-	<div class="lemma-widget"><ul class="lemma-widget_lemma-list"><li class="lemma-widget_lemma"><span class="lemma-widget_lemma-value">lemma1</span> <span class="lemma-widget_lemma-language">en</span></li><li class="lemma-widget_lemma"><span class="lemma-widget_lemma-value">lemma2</span> <span class="lemma-widget_lemma-language">fr</span></li></ul> <div class="lemma-widget_controls"><button type="button" class="lemma-widget_control">wikibase-edit</button> <!----> <!----></div></div>
+	<div class="lemma-widget"><ul class="lemma-widget_lemma-list"><li class="lemma-widget_lemma"><span class="lemma-widget_lemma-value">lemma1</span> <span class="lemma-widget_lemma-language">en</span></li><li class="lemma-widget_lemma"><span class="lemma-widget_lemma-value">l&#xE9;mma2</span> <span class="lemma-widget_lemma-language">fr</span></li><li class="lemma-widget_lemma"><span class="lemma-widget_lemma-value">&#xAE30;&#xBCF8;&#xD615;</span> <span class="lemma-widget_lemma-language">ko</span></li><li class="lemma-widget_lemma"><span class="lemma-widget_lemma-value">&#x41B;&#x435;&#x43C;&#x43C;&#x430;</span> <span class="lemma-widget_lemma-language">ru</span></li><li class="lemma-widget_lemma"><span class="lemma-widget_lemma-value">&#x644;&#x644;&#x64A;&#x645;&#x627;</span> <span class="lemma-widget_lemma-language">ar</span></li></ul> <div class="lemma-widget_controls"><button type="button" class="lemma-widget_control">wikibase-edit</button> <!----> <!----></div></div>
```

Comment:
So this is really the outcome of `npm run-script populate-fixtures` - and it contains html entities for characters that, in an UTF-8 world could be expressed without html entities. In the clients we do not want this to happen (right?!) so why do we have a test proving the opposite?